File: Java/FizzBuzz/src/FizzBuzz.java
public class FizzBuzz {

  public FizzBuzz(int max) {
    for (int i = 1; i <= max; i++) {
      if (i % 3 == 0) { System.out.print("Fizz"); }
      if (i % 5 == 0) { System.out.print("Buzz"); }
      if (i % 5 != 0 && i % 3 != 0) { System.out.print(i); }
      System.out.println();
    }
  }
  
}

File: Pascal/FizzBuzz/fizzbuzz.pas
program fizzbuzz;

var i: integer;
begin
  for i := 1 to 100 do begin
    if (i mod 15) = 0 then
      writeln('FizzBuzz')
    else if (i mod 5) = 0 then
      writeln('Buzz')
    else if (i mod 3) = 0 then
      writeln('Fizz')
    else
      writeln(i)
  end
end.